main.go

```diff
@@ -12,13 +12,9 @@ import (
 	"syscall"
 
 	"github.com/zarf-dev/zarf/src/cmd"
-	"github.com/zarf-dev/zarf/src/config"
 	"github.com/zarf-dev/zarf/src/pkg/lint"
 )
 
-//go:embed cosign.pub
-var cosignPublicKey string
```

Comment:
IIRC this was used before the transition from defenseunicorns to zarf-dev to check packages from the defense unicorn GitHub org. We removed this check, but didn't remove the variable, which is currently not doing anything